package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gigalixir-action",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Can you revert the version change? I will do this myself when i cut the next release.

Reply:
👍